api/src/main/java/org/pmiops/workbench/workspaces/WorkspacesController.java

```diff
@@ -326,29 +314,7 @@ private FirecloudWorkspaceId generateFirecloudWorkspaceId(String namespace, Stri
   @Override
   public ResponseEntity<WorkspaceResponse> getWorkspace(
       String workspaceNamespace, String workspaceId) {
-    org.pmiops.workbench.db.model.Workspace dbWorkspace =
-        workspaceService.getRequired(workspaceNamespace, workspaceId);
-
-    org.pmiops.workbench.firecloud.model.WorkspaceResponse fcResponse;
-    org.pmiops.workbench.firecloud.model.Workspace fcWorkspace;
-
-    WorkspaceResponse response = new WorkspaceResponse();
-
-    // This enforces access controls.
-    fcResponse = fireCloudService.getWorkspace(workspaceNamespace, workspaceId);
-    fcWorkspace = fcResponse.getWorkspace();
-
-    if (fcResponse.getAccessLevel().equals(WorkspaceService.PROJECT_OWNER_ACCESS_LEVEL)) {
-      // We don't expose PROJECT_OWNER in our API; just use OWNER.
-      response.setAccessLevel(WorkspaceAccessLevel.OWNER);
-    } else {
-      response.setAccessLevel(WorkspaceAccessLevel.fromValue(fcResponse.getAccessLevel()));
-      if (response.getAccessLevel() == null) {
-        throw new ServerErrorException("Unsupported access level: " + fcResponse.getAccessLevel());
-      }
-    }
-    response.setWorkspace(workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace));
-    return ResponseEntity.ok(response);
+    return ResponseEntity.ok(workspaceService.getWorkspace(workspaceNamespace, workspaceId));
```

Comment:
OK, so this will just convert to a 500 if you throw inside `getWorkspace()`?

Reply:
Should be. This line shouldn't execute if an exception is thrown inside of the `getWorkspace()` call. Some error handler should catch the exception and generate a 500.